src/orquestra/sdk/_base/cli/_ui/_presenters.py

```diff
@@ -362,19 +362,51 @@ def show_wf_run(self, summary: ui_models.WFRunSummary):
         self._console.print(self.get_wf_run(summary))
 
     def show_wf_list(self, summary: ui_models.WFList):
+        """Display a list of workflow runs.
+
+        The list is shown as a table with the following headings:
+        - Workflow Run ID
+        - Status
+        - Succeeded Tasks
+        - Start Time
+        - [Owner]
+
+        The 'Owner' column is displayed as long as at least one run in the list has a
+        populated ``owner`` field.
+
+        Example output::
+
+            Workflow Run ID   Status     Succeeded Tasks   Start Time                 Owner
+            ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+            wf1               SUCCEEDED  1/1               Fri Feb 24 08:26:07 2023   evil/emiliano.zapata@zapatacomputing…
+            wf2               WAITING    0/1                                          evil/emiliano.zapata@zapatacomputing…
+
+        Args:
+            summary: A list of workflow run summaries to be displayed.
+        """  # noqa: E501
+        show_owner: bool = False
+        for row in summary.wf_rows:
+            if row.owner:
+                show_owner = True
+                break
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking, style):** If we want to use fancy one-liners xD
```suggestion
        show_owner = any(row.owner for row in summary.wf_rows)
```